templates/archive-meetings.php

```diff
@@ -668,4 +673,12 @@ public function end_el(&$output, $item, $depth = 0, $args = [])
 </script>
 
 <?php
-get_footer();
+if (function_exists('wp_is_block_theme') && wp_is_block_theme()) {
+   echo do_blocks('<!-- wp:template-part {"slug":"footer","theme":"twentytwentytwo","tagName":"footer","className":"site-footer","layout":{"inherit":true}} /-->');
```

Comment:
looks like this adds functionality specific to the twenty twenty two theme. what happens if it's a block theme but not twenty twenty two? should we be checking if that theme is installed? or can we think of a solution for block themes that doesn't require twenty twenty two?

Reply:
My understanding is that the requirement for a unique theme name in the template-part function call is considered a bug in the 5.9 code base, and that a fix is expected from the Gutenberg team. Just don't know when that will be at this point. We will likely need to address when a fix comes out.